src/locale/lang/tr.js

```diff
@@ -172,6 +172,31 @@ export default {
       exit: 'Exit',
       minimizeToTray: 'Küçült',
     },
+    unm: {
+      enable: 'Enable',
+      audioSource: {
+        title: 'Audio Source',
+      },
+      enableFlac: {
+        title: 'Enable FLAC',
+        desc: 'Need to clear the cache to take effect after enable',
```

Comment:
To take effect, it may be required to clear the cache after enabling this function.